### PR TITLE
Log send command

### PR DESCRIPTION
**Pull request by patrickxb**
This adds `keyabse log send`.

It works with keybase/keybase#coyne/CORE-2423.

r? @malgorithms @maxtaco 

(for testing with devel build, I linked my production log files to the devel home directory locations reported by `status` command, since with devel the stdout isn't saved to files.)

### CODE REVIEW

**Comment by malgorithms** [comment]
LGTM although I don't know Go.

**Comment by patrickxb** [comment]
I added confirmation and github url.

File: go/client/cmd_log_send.go [hunk 1]
```diff
@@ -0,0 +1,202 @@
...
+}
+
+func (c *CmdLogSend) Run() error {
+	if err := c.confirm(); err != nil {
+		return err
+	}
+
+	c.G().Log.Debug("getting keybase status")
+	statusCmd := &CmdStatus{Contextified: libkb.NewContextified(c.G())}
+	status, err := statusCmd.load()
```
Comment (maxtaco) [inline go/client/cmd_log_send.go]: This gets extended status?

File: go/client/cmd_log_send.go [hunk 2]
```diff
@@ -0,0 +1,202 @@
...
+}
+
+func (c *CmdLogSend) confirm() error {
+	ui := c.G().UI.GetTerminalUI()
+	ui.Printf("This command will send recent keybase log entries to keybase.io\n")
+	ui.Printf("for debugging purposes only.\n\n")
+	ui.Printf("These logs don’t include your private keys or encrypted data,\n")
+	ui.Printf("but they will include filenames and other metadata keybase normally\n")
+	ui.Printf("can’t read, for debugging purposes.\n")
+	return ui.PromptForConfirmation("Continue sending logs to keybase.io?")
```
Comment (maxtaco) [inline go/client/cmd_log_send.go]: Make sure you default to "no"!

Comment (patrickxb) [inline go/client/cmd_log_send.go]: Continue sending logs to keybase.io? (type 'YES' to confirm):

File: go/client/cmd_log_send.go (hunk 1 above)
Comment (patrickxb) [inline go/client/cmd_log_send.go]: yes, it gets everything that the status command gets.

**Comment by maxtaco** [comment]
:+1:

**Pull request merged**